.github/NativeBuildReport.java

```diff
@@ -0,0 +1,95 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
```

Comment:
This was copied from the script we have in Quarkus proper

Reply:
We should move this class to https://github.com/quarkusio/jbang-catalog at some point

Reply:
A central shortcut catalog is not suitable for hosting scripts to do releases. You want the release scripts tag/versioned together with your artifacts.

You can add a catalog file to this Repo if you want to simplify its usage but I would highly recommend to not move these scripts to central catalog.